### PR TITLE
feature/AddCSSListing add black description option

### PR DESCRIPTION
**Pull request by HanqingZhou**
Add black description option:
Original description:
Screen Shot 2022-05-17 at 5 07 54 PM
Black description:
Screen Shot 2022-05-17 at 5 07 38 PM

### CODE REVIEW

File: src/blocks/Listing/Listing.scss [hunk 1]
```diff
@@ -39,6 +39,10 @@
             color: $link;
         }
 
+        p {
+            color: black;
+        }
```
Comment (zahraard) [inline src/blocks/Listing/Listing.scss:44]: please use the variable color, 
```suggestion
        color: $black;
```

**Comment by comichelle** [comment]
Other than Zahra's comment. Looks good to me.

File: src/blocks/Listing/Listing.scss (hunk 1 above)
Comment (HanqingZhou) [reply src/blocks/Listing/Listing.scss:44]: Thank you @zahraard changed to $black variable color

**Comment by HanqingZhou** [comment]
> Other than Zahra's comment. Looks good to me.

Thank you @comichelle

**Review by zahraard — APPROVED** [review]
Thanks, please merge it.

**Pull request merged**